src/notifications/types/types.tsx

```diff
@@ -0,0 +1,38 @@
+import { DateTime } from "luxon";
+
+interface UserInputParams {
+    email: string;
+    carNickname: string;
+    notificationDate: DateTime | null;
+  }
+  
+  export class UserInput {
```

Comment:
NIT: I feel like creating a class for this might not be necessary... 🤔 It feels like it is just a typed object.

Reply:
My teacher in school loved using classes everywhere in React, but after a bit of reading I'm on your side! It seems a little overengineered to me.